lib/blockchain.go

```diff
@@ -1976,7 +1976,7 @@ func (bc *Blockchain) processHeaderPoW(blockHeader *MsgDeSoHeader, headerHash *B
 }
 
 // ProcessHeader is a wrapper around processHeaderPoW and processHeaderPoS, which do the leg-work.
-func (bc *Blockchain) ProcessHeader(blockHeader *MsgDeSoHeader, headerHash *BlockHash) (_isMainChain bool, _isOrphan bool, _err error) {
+func (bc *Blockchain) ProcessHeader(blockHeader *MsgDeSoHeader, headerHash *BlockHash, verifySignatures bool) (_isMainChain bool, _isOrphan bool, _err error) {
```

Comment:
After testing the syncing on devnet, header syncing is actually quite fast. it connects the full 2k header bundle in 1-2 seconds. It's unlikely for header syncing to benefit from the verifySignatures flag. If the flag isn't used for the PoW header syncing, maybe it's fine to exclude it for PoS too. Just FYI